assets/Product.wxs

```diff
@@ -43,9 +43,9 @@
     <!-- Default value of Checkbox of starting PowerShell after installation -->
     <Property Id="WixShellExecTarget" Value="[$(var.ProductVersionWithName)]pwsh.exe"/>
     <CustomAction Id="LaunchApplication" BinaryKey="WixCA" DllEntry="WixShellExec" Impersonate="yes" />
-    <CustomAction Id="SaveShortcutPath" Property="ShortcutPath" Value="[$ApplicationProgramsMenuShortcut]$(var.ProductSemanticVersionWithNameAndOptionalArchitecture).lnk" />
-    <CustomAction Id="FixShortcutWorkingDirectory" Script="vbscript" HideTarget="no" Impersonate="no">
-      shortcutPath = Session.Property("ShortcutPath")
+    <CustomAction Id="SaveShortcutPath" Property="FixShortcutWorkingDirectory" Value="[$ApplicationProgramsMenuShortcut]$(var.ProductSemanticVersionWithNameAndOptionalArchitecture).lnk" />
```

Comment:
Do we really need rename `ShortcutPath` to `FixShortcutWorkingDirectory`?

Reply:
The property name must match the ID of the custom action in order to populate CustomActionData.

https://stackoverflow.com/questions/11233267/how-to-pass-customactiondata-to-a-customaction-using-wix

Reply:
Thanks for clarify! It is not obvious. Please add the comment with the link.

Reply:
I know that WiX is not self-explanatory or intuitive to learn but do we really need to document basic WiX stuff here?

Reply:
Somebody can change the name. We haven't  msi package tests. So I'd prefer a protective comment.

Reply:
Also I'd use `SetShortcutWorkingDirectory` name.